### PR TITLE
Iotbzh/julien/clean soc

### PR DESCRIPTION
**Pull request by julien-massot**

### CODE REVIEW

File: boards/arm/rcar_ulcb/Kconfig.board [hunk 1]
```diff
@@ -3,4 +3,4 @@
 
 config BOARD_RCAR_ULCB_CR7
 	bool "Cortex-R7 for Renesas M3ULCB/H3ULCB"
-	depends on SOC_RENESAS_RCAR_CR7
+	depends on SOC_R8A77950
```
Comment (aaillet) [inline boards/arm/rcar_ulcb/Kconfig.board]: rename to 951

File: boards/arm/rcar_ulcb/rcar_ulcb_cr7.dts [hunk 2]
```diff
@@ -6,10 +6,10 @@
  */
 
 /dts-v1/;
-#include <arm/renesas/rcar_cr7.dtsi>
+#include <arm/renesas/gen3/r8a77950.dtsi>
```
Comment (aaillet) [inline boards/arm/rcar_ulcb/rcar_ulcb_cr7.dts]: 951

File: boards/arm/rcar_ulcb/Kconfig.board [hunk 3]
```diff
@@ -3,4 +3,4 @@
 
 config BOARD_RCAR_ULCB_CR7
 	bool "Cortex-R7 for Renesas M3ULCB/H3ULCB"
```
Comment (aaillet) [inline boards/arm/rcar_ulcb/Kconfig.board]: h3 only

File: boards/arm/rcar_ulcb/rcar_ulcb_cr7_defconfig [hunk 4]
```diff
@@ -1,4 +1,5 @@
-CONFIG_SOC_RENESAS_RCAR_CR7=y
+CONFIG_SOC_R8A77950=y
```
Comment (aaillet) [inline boards/arm/rcar_ulcb/rcar_ulcb_cr7_defconfig]: 951

File: soc/arm/renesas_rcar/gen3/Kconfig.defconfig.r8a77950 [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 IoT.bzh
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_R8A77950
```
Comment (aaillet) [inline soc/arm/renesas_rcar/gen3/Kconfig.defconfig.r8a77950]: 951

File: soc/arm/renesas_rcar/gen3/Kconfig.defconfig.r8a77950 [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+
+config SOC
+       default "r8a77950"
```
Comment (aaillet) [inline soc/arm/renesas_rcar/gen3/Kconfig.defconfig.r8a77950]: 951

File: soc/arm/renesas_rcar/gen3/Kconfig.defconfig.r8a77950 [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+
+endif # SOC_R8A77950
```
Comment (aaillet) [inline soc/arm/renesas_rcar/gen3/Kconfig.defconfig.r8a77950]: 951

File: soc/arm/renesas_rcar/gen3/Kconfig.soc [hunk 8]
```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2020 IoT.bzh
+# SPDX-License-Identifier: Apache-2.0
+
+choice
+	prompt "Renesas RCar SoC Selection"
+	depends on SOC_SERIES_RCAR_GEN3
+
+config SOC_R8A77950
+	bool "R8A77950"
+
```
Comment (aaillet) [inline soc/arm/renesas_rcar/gen3/Kconfig.soc]: 951

File: boards/arm/rcar_ulcb/Kconfig.board (hunk 3 above)
Comment (julien-massot) [reply boards/arm/rcar_ulcb/Kconfig.board]: thanks fixed

File: boards/arm/rcar_ulcb/Kconfig.board (hunk 1 above)
Comment (julien-massot) [reply boards/arm/rcar_ulcb/Kconfig.board]: thanks fixed

**Review by aaillet — APPROVED** [review]

**Pull request merged**